src/style/app-fixed.module.css

```
@@ -9152,6 +9566,50 @@ form {
   background-color: var(--people-card-container-bg);
 }
 
+.userContactDetailPersonalCardHeader {
+  background-color: #a8c7fa;
+  color: #555;
+  background-color: #a8c7fa;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)